iina/es.lproj/KeyBinding.strings

```diff
@@ -0,0 +1,110 @@
+/* 
+  KeyBinding.strings
+  iina
+
+  Created by lhc on 4/2/2017.
+  Copyright © 2017 lhc. All rights reserved.
+*/
+
+"cmd.ignore" = "Ignorar";
+"cmd.seek" = "Buscar";
+"cmd.revert-seek" = "Revertir buscar";
+"cmd.set" = "Establecer";
+"cmd.cycle" = "Alternar";
+"cmd.cycle-values" = "Alternar entre";
+"cmd.add" = "Cambiar";
+"cmd.multiply" = "Multiplicar";
+"cmd.frame-step" = "Fotograma siguiente";
+"cmd.frame-back-step" = "Fotograma anterior";
+"cmd.ab-loop" = "Bucle A-B";
+"cmd.sub-seek" = "Buscar subs";
+"cmd.playlist-next" = "Siguiente";
+"cmd.playlist-prev" = "Anterior";
+"cmd.playlist-move" = "Mover elemento de la lista de reproducción";
+"cmd.playlist-clear" = "Borrar lista de reproducción";
+"cmd.playlist-remove" = "Eliminar actual";
+"cmd.playlist-shuffle" = "Aleatorio";
+"cmd.write-watch-later-config" = "Record play history";
+"cmd.stop" = "Detener";
+"cmd.quit" = "Salir";
+"cmd.quit-watch-later" = "Salir (guardar progreso)";
+"cmd.screenshot" = "Captura de pantalla";
+"cmd.screenshot-to-file" = "Screenshot to file";
+"cmd.loadfile" = "Carga Archivo";
+"cmd.loadlist" = "Cargar lista de reproducción";
+"cmd.run" = "Ejecutar";
+
+"iina.flip" = "Voltear";
+"iina.mirror" = "Reflejar";
+
+"iina.toggle-pip" = "Alternar Picture-in-Picture";
+"iina.open-file" = "Abrir archivo";
+"iina.open-url" = "Abrir URL";
+"iina.video-panel" = "Panel de video";
+"iina.audio-panel" = "Panel de audio";
+"iina.sub-panel" = "Panel de subtítulos";
+"iina.playlist-panel" = "Panel de listas de reproducción";
+"iina.chapter-panel" = "Panel de capítulos";
+"iina.toggle-flip" = "Alternar volteo";
+"iina.toggle-mirror" = "Alternar reflejo";
+"iina.save-playlist" = "Guardar la lista de reproducción actual";
+"iina.delete-current-file" = "Eliminar archivo actual";
+"iina.find-online-subs" = "Buscar subtítulos en línea";
+"iina.save-downloaded-sub" = "Guardar subtítulos descargados";
+
+"yes" = "Sí";
+"no" = "No";
+"to" = "a";
+"by" = "para";
+"in" = "en";
+"add" = "incrementar";
+"minus" = "disminuir";
+"forward" = "adelantar";
+"backward" = "restroceder";
+"seek-to" = "to";
+
+"relative" = "s";
+"absolute" = "s";
+"absolute-percent" = "%";
+"relative-percent" = "%";
+"relative+exact" = "s (exact)";
```

Comment:
@lhc70000 Should exact be translated here? It is in some translations, but not in others.